.travis.yml

```diff
@@ -6,6 +6,7 @@ env:
   - RUSTFLAGS='-D warnings'
 
 rust:
+  - 1.35.0
   - 1.36.0
```

Comment:
Might as well bump this to 1.37 now.

Reply:
Or remove it, since we already have `stable`.

Reply:
Ye, building against stable and nightly should be enough. I just added 1.35.0 to check for backwards compatability. Do you have any idea why the windows builds fail in travis?

Reply:
Actually I just meant remove 1.36 and only keep 1.35. No idea why the windows builds are failing. They are generally very unreliable, I'll move this to Azure Pipelines when I get the chance.

Reply:
Maybe going back to the oldest rust version u intend to support would be a good choice for the CI (despite using stable/latest).

Reply:
I didn't intend to support anything older than 1.36, but your fix is simple enough, so going back to 1.35 is fine.